javascripts/controllers/NewCtrl.js

```diff
@@ -1,48 +1,54 @@
 'use strict';
 
 app.controller("NewCtrl", function($location, $rootScope, $scope, ContactService) {
-	$scope.controller = "Hello, NewCtrl. Route:/contacts/new";
-	// console.log("Hello, NewCtrl. Route:/contacts/new");
-
-
-	// const createContact = (contact) => {
-	// 	return {
-	// 		"firstName": contact.firstName,
-	// 		"lastName":  contact.lastName,
-	// 		"phoneHome": contact.phoneHome,
-	// 		"phoneWork": contact.phoneWork,
-	// 		"emailHome": contact.emailHome,
-	// 		"emailWork": contact.emailWork,
-	// 		"notes":	 contact.notes,
-	// 		"uid": $rootScope.uid
-	// 	};
-	// };
+	if($rootScope.flag) {
+		$scope.controller = "Hello, NewCtrl. Route:/contacts/edit/:id";
```

Comment:
At this point, you don't need to have these "hello" messages print in the partials. Clever way of doing this, though!

Reply:
Fixed